public/locales/en/translation.json

```diff
@@ -1070,6 +1070,12 @@
         }
       }
     },
+    "gatewayForms": {
+      "Filepicker": {
```

Comment:
I would remove gatewayForms here, since it can also be used elsewhere and "Filepicker" should be "filepicker"